src/main/flight/mixer_init.c

```diff
@@ -40,6 +40,8 @@
 #include "flight/mixer_tricopter.h"
 #include "flight/pid.h"
 
+#include "mixer.h"
```

Comment:
`mixer.h` goes in front of `mixer_tricopter.h`
```suggestion
#include "flight/mixer.h"
#include "flight/mixer_tricopter.h"
#include "flight/pid.h"
```